app.js

```diff
@@ -0,0 +1,100 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const bodyParser = require("body-parser");
+const School = require("./School")
+const Student = require('./Student')
+const port = 3000;
+
+let mySchool = new School();
+// let approvedMessage = {message: "Created a new class", timestamp: "Todays Date"}
+
+app.use(cors());
+
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(bodyParser.json());
+
+app.post("/class", (req, res) => {
+    const { name, teacher } = req.body
+    // console.log(req.body);
+    let keys = Object.keys(mySchool.classes)
+    // console.log("my keys",keys)
+    if(keys.includes(name) || name === "" || teacher === "") {
+        console.log("Please fill out all the information or Class already exists");
+        
+        res.json({
+            error: "Please fill out all the information or Class already exists",
+            timestamp: "YYYY, MM/DD HH:MM:SS"
+        })
+    }else {
+        let newClass = mySchool.addClass(req.body.name, req.body.teacher)
+        res.json({
+            newClass,
+            message: "Created a new class",
+            timestamp: "YYYY, MM/DD HH:MM:SS"
```

Comment:
This is just a string, using the Date class then turning it into a string would work perfectly in this situiation